test/test_hpack.py

```diff
@@ -653,6 +654,30 @@ def test_can_decode_multiple_header_table_size_changes(self):
 
         assert d.decode(data) == expect
 
+    def test_header_table_size_change_above_maximum(self):
+        """
+        If a header table size change is received that exceeds the maximum
+        allowed table size, it is rejected.
+        """
+        d = Decoder()
+        d.max_allowed_table_size = 127
+        data = b'?a\x82\x87\x84A\x8a\x08\x9d\\\x0b\x81p\xdcy\xa6\x99'
```

Comment:
I have a bit of a pet peeve that these blocks of binary data are somewhat opaque to somebody reading the tests. Would there be any value in breaking them up and explaining what different parts of the blob are/do?